openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -243,6 +243,19 @@ message SKAdNetworkRequest {
 message ImpExt {
   // Publisher's SKAdNetwork information.
   optional SKAdNetworkRequest skadn = 1;
+
+  // To differentiate between impression opportunities
+  // that support Protected Audience API on-device auction from
+  // those that only support traditional server-side exchange auction
+  enum AuctionEnvironment {
+    // Traditional server-side auctions
+    SERVER_SIDE_AUCTION = 0;
+    // Requests with Protected Audience API support,
+    // in which a contextual auction runs on the exchange's servers
+    // and the interest group bidding and the final auction runs in the browser
+    ON_DEVICE_INTEREST_GROUP_AUCTION = 1;
+  }
+  optional AuctionEnvironment ae = 2 [default = SERVER_SIDE_AUCTION];
```

Comment:
Integer (or boolean) type only please. The enums can be defined for convenience/reference, though we do not use them directly in the spec.